File: app/src/main/java/org/kikermo/thingsaudioreceiver/model/net/rest/RestCallback.java
package org.kikermo.thingsaudioreceiver.model.net.rest;

import org.kikermo.thingsaudioreceiver.model.data.Track;

import java.util.List;

/**
 * Created by EnriqueR on 28/02/2017.
 */

public interface RestCallback {

   void skipNextReceived();

    void skipPrevReceived();

    void listReceived(List<Track> trackList);

    void playReceived();

    void pauseReceived();
}

File: app/src/main/java/org/kikermo/thingsaudioreceiver/model/net/rest/RestCallback.kt
package org.kikermo.thingsaudioreceiver.model.net.rest

import org.kikermo.thingsaudio.core.model.Track

interface RestCallback {

    fun skipNextReceived()

    fun skipPrevReceived()

    fun listReceived(trackList: List<Track>)

    fun playReceived()

    fun pauseReceived()
}
